src/js/share/share.js

```diff
@@ -188,10 +189,11 @@ export class Share {
     });
 
     if (sceneName) {
-      sceneHeader.innerHTML = `${sceneName}<i class="material-icons">expand_more</i>`;
+      sceneHeader.innerHTML = `<div style="font-weight: normal;">${sceneName}</div><i class="material-icons">chevron_left</i>`;
```

Comment:
- when starting, scenes are collapsed so we use "chevron_left" (<) instead of "expand_more" (v)-
- also font-style is normal on start up